RecoEgamma/EgammaElectronProducers/plugins/GsfElectronBaseProducer.h

```diff
@@ -59,6 +59,10 @@ class GsfElectronBaseProducer : public edm::stream::EDProducer< edm::GlobalCache
     const GsfElectronAlgo::CutsConfiguration cutsCfgPflow_ ;
     ElectronHcalHelper::Configuration hcalCfg_ ;
     ElectronHcalHelper::Configuration hcalCfgPflow_ ;
+
+    // used to make some provenance checks
+    edm::EDGetTokenT<edm::ValueMap<float>> pfMVA_;
```

Comment:
Sorry I had to update this PR because I realized I removed the "pfMvaTag" and it's usage in the GEDGsfElectronProducer prematurely before. I was confused because it was not used in the actual algo (which previously owned it via a config struct), but just for some provenance checks in the producer. Now, it's not passed on to the algo anymore at least.